test/integration/targets/any_errors_fatal/dynamic_playbook.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: all
+  gather_facts: false
+  any_errors_fatal: "{{ not skip_errors }}"
```

Comment:
also needs a test in which its set to a templated value that is not a valid boolean

Reply:
added